vuepress/docs/v7.0/tutorials/devops/external-id-management.md

```diff
@@ -2,75 +2,83 @@
 -   The Keycloak admin password, e.g. password123
+-   The base URL for the Keycloak server, including the auth value, e.g. https://YOUR-KEYCLOAK-INSTANCE.com/auth
 
--   The base url for the Keycloak server, including the auth value, e.g.
-    https://your-keycloak-instance.com/auth
+### 2. Generate the Secret
 
-### 2. Generate the secret
-
-You now need to generate a secret with name *keycloak-admin-secret*
-using the information retrieved from step 1. The Entando administrator
-will automatically detect this secret by name, and use it to log onto
-the provided Keycloak server.
-
-Here is an example of the secret you will need to construct:
+Generate a Secret named *keycloak-admin-secret* with the information retrieved in Step 1. For example:
 
     ---
     apiVersion: v1
     stringData:
         username: #the username of the Keycloak admin user for the "entando" realm
-        password: #the password of this Keycloak admin user
-        url: #the base url of the Keycloak service, typically ending with the path /auth
+        password: #the password of the Keycloak admin user
+        url: #the base URL of the Keycloak service, typically ending with the path /auth
     kind: Secret
     metadata
         name: keycloak-admin-secret
         namespace: YOUR-APP-NAMESPACE
     type: Opaque
 
+The client for the admin user will automatically detect this Secret by name and use it to log in to the Keycloak server at the specified URL.
+
 > **Note**
->
-> To encode your values, in bash, you can do
-> `echo YOUR-VALUE | base64`
+> To encode a value in bash, use `echo YOUR-SECRET-VALUE | base64`
+
+### 3. Upload the Secret
+
+Upload the Secret to the namespace where you want to deploy your Entando instance:
+
+    oc create -f YOUR-SECRET.yaml -n YOUR-NAMESPACE
+
+### 4. Create a YAML configuration file
+
+Create a YAML file to configure Keycloak, based on the following template:
+
+```
+apiVersion: entando.org/v1
+kind: EntandoKeycloakServer
+metadata:
+  name: external-keycloak
+  namespace: {{ namespace }}
+spec:
+  environmentVariables: []
+  provisioningStrategy: UseExternal
+  adminSecretName: keycloak-admin-secret
+  frontEndUrl: >-
+    http://<keycloak_url>/auth
+```
+
+### 5. Apply the YAML configuration file
 
-### 3. Upload the secret
+Apply the YAML configuration file to the namespace where you want to deploy your Entando instance:
 
-Next upload the secret to the namespace where you want to deploy your
-Entando instance.
+```
+kubectl apply -f YOUR-YAML-FILE.yaml -n YOUR-NAMESPACE
 
-    oc create -f YOUR-SECRET.yaml -n MY-APP-NAMESPACE
+```
 
-### 4. Deploy the Entando application
+### 6. Deploy the Entando Application
 
-Now you are ready to deploy your Entando application and the
-administrator will reuse the *keycloak-admin-secret* secret to populate
-the environment correctly.
+You are now ready to deploy your Entando Application. The admin user clien will reuse *keycloak-admin-secret* to populate the environment correctly.
```

Comment:
I would still include the article - the keycloak-admin-secret since it refers to that specific secret.